scripts/generate_test_sql/events.py

```diff
@@ -3,10 +3,10 @@
 
 
 FEEDING_EVENT_NAMES_LIST = (
-    "Breakfast",
-    "Brunch",
-    "Lunch",
-    "Dinner",
+    'Breakfast',
```

Comment:
Why change this unrelated file? Was the linter complaining, or was it some other reason?

Reply:
Yeah this was the linter's doing - should I revert the change?